CHANGES.md

```diff
@@ -5,6 +5,7 @@
 ##### Fixes :wrench:
 
 - Fixed an issue where `Camera.flyTo` would not work properly with a non-WGS84 Ellipsoid. [#9498](https://github.com/CesiumGS/cesium/pull/9498)
+- Fixed an issue that the viewport is not applied to `ViewportQuad` after setting the width and height of the rectangle.[#9511](https://github.com/CesiumGS/cesium/pull/9511)
```

Comment:
@bbbbx - The spec looks good! Just a minor nitpick about wording and I think this is good to go -
```suggestion
- Fixed an issue where setting the `ViewportQuad` rectangle after after creating the viewport had no effect.[#9511](https://github.com/CesiumGS/cesium/pull/9511)
```